tldraw/packages/core/src/lib/tools/TLSelectTool/states/IdleState.ts

```diff
@@ -43,6 +43,11 @@ export class IdleState<
       inputs: { ctrlKey },
     } = this.app
 
+    if (event.button === 2) {
```

Comment:
May consider adding CTRL (or alt on non-Mac?) + left click for context menu later

Reply:
Alt is working, but it could also trigger the native context menu.
I need to fix that.

Reply:
I guess `alt` clicking is handled by Radix UI?